eng/testing/AppleRunnerTemplate.sh

```diff
@@ -39,14 +35,17 @@ while true; do
     fi
 done
 
-XHARNESS_OUT="$EXECUTION_DIR/xharness-output"
+XCODE_PATH="`xcode-select -p`/../.."
+export XHARNESS_OUT="$EXECUTION_DIR/xharness-output"
 
-dotnet xharness ios test --app="$APP_BUNDLE" \
-    --targets=$TARGET \
+dotnet xharness ios test  \
+    --targets="$TARGET"   \
+    --app="$APP_BUNDLE"   \
+    --xcode="$XCODE_PATH" \
     --output-directory=$XHARNESS_OUT
 
 _exitCode=$?
 
-echo "Xharness artifacts: $XHARNESS_OUT"
+echo "XHarness artifacts: `ls -lh $XHARNESS_OUT`"
```

Comment:
I feel like I commented about this in some PR already but I don't like the `ls` here. It means I can't cmd+click the path in my terminal to open the folder anymore since the path is no longer printed.

Reply:
Changed